validation/certs/utils/io.py

```diff
@@ -24,9 +24,8 @@ def export_chain(chain, filename):
 
 def export_crl(crl_der, filename):
     os.makedirs(os.path.dirname(filename), exist_ok=True)
-    crl_pem = _der_to_pem(crl_der, CRL_LABEL)
-    with open(filename, "w") as file:
-        file.write(crl_pem)
+    with open(filename , "bw") as file:
```

Comment:
I was wondering why you switched to DER but it's true that it's easier to process and I haven't yet seen a PEM encoded CRL in the wild...

Reply:
Should not be that much of a problem.